src/Form/AbstractFormBuilder.php

```diff
@@ -179,11 +179,17 @@ private function buildComponent(array $attributes): string
 					break;
 			}
 
+			$id = $attributes["{$component}Id"] ?? '';
+
 			// Loop children and do the same ad on top level.
-			foreach ($attributes[$key] as $item) {
+			foreach ($attributes[$key] as $innerKey => $item) {
 				// Determin component name.
 				$innercComponent = $item['component'] ? HelpersComponents::kebabToCamelCase($item['component']) : '';
 
+				if ($key === 'radiosContent') {
+					$item["{$innercComponent}Id"] = "{$id}[{$innerKey}]";
```

Comment:
The `$id` will fallback to empty string in case the `$attributes["{$component}Id"]` is not set or empty. In this case, the right-hand side will throw an error since you're trying to get a key out of a string.

Or am I missing something?

Reply:
I see the confusion but this will not actually get any thing but rather output this kind of string blablaSomeID[0], blablaSomeID[1], etc for the radio name attribute something like this https://stackoverflow.com/questions/11624871/radiobutton-array-html

Reply:
Oh! I see, I omitted the `{}` around the ID. Got it 👍🏼